dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -187,6 +190,7 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
         {
           committed: Pd::Application::Teacher1819Application.options[:committed].last,
           able_to_attend_single: NO,
+          able_to_attend_multiple: Teacher1819ApplicationConstants::TEXT_FIELDS[:no_explain],
```

Comment:
note, you can `include Teacher1819ApplicationConstants` in the test and then reference `TEXT_FIELDS` directly. Either way is fine.